From: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Subject: ARM: shmobile: porter: add CAN0 DT support

Define the Porter board dependent part of the CAN0 device node.

This patch is  analogous to the commit 457acc4a6ce8 (ARM: shmobile:
henninger: add CAN0 DT support) as there are no differences between the
boards in this respect.

Signed-off-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>

---
 arch/arm/boot/dts/r8a7791-porter.dts |   12 ++++++++++++
 1 file changed, 12 insertions(+)

Index: linux-stable/arch/arm/boot/dts/r8a7791-porter.dts
===================================================================
--- linux-stable.orig/arch/arm/boot/dts/r8a7791-porter.dts
+++ linux-stable/arch/arm/boot/dts/r8a7791-porter.dts
@@ -134,6 +134,11 @@
 		renesas,groups = "vin0_data8", "vin0_clk";
 		renesas,function = "vin0";
 	};
+
+	can0_pins: can0 {
+		renesas,groups = "can0_data";
+		renesas,function = "can0";
+	};
 };
 
 &scif0 {
@@ -287,3 +292,10 @@
 &pciec {
 	status = "okay";
 };
+
+&can0 {
+	pinctrl-0 = <&can0_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+};
